src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp

```diff
@@ -1147,6 +1149,25 @@ class spell_kiljaeden_dragon_breath : public SpellScript
     }
 };
 
+class spell_kiljaeden_breath_haste : public SpellScript
+{
+    PrepareSpellScript(spell_kiljaeden_breath_haste);
```

Comment:
```suggestion
    PrepareSpellScript(spell_kiljaeden_breath_haste);

    bool Validate(SpellInfo const* /*spellInfo*/) override
    {
        return ValidateSpellInfo({ SPELL_BREATH_HASTE_EFFECT });
    }
```

https://www.azerothcore.org/wiki/core-scripts#spell-scripts

> Spells used within the SpellScript should always be validated at the top of the script.

Reply:
Removed cpp with sql change